roles/matrix-synapse/templates/synapse/homeserver.yaml.j2

```diff
@@ -1647,7 +1647,7 @@ room_prejoin_state:
 
 # A list of application service config files to use
 #
-app_service_config_files: {{ matrix_synapse_app_service_config_files|to_json }}
+app_service_config_files: {{ matrix_synapse_config_files|to_json }}
```

Comment:
`matrix_synapse_config_files` seems like an odd and inconsistent name.

We try to make variable names match the setting that they control. In this case, we can't use `matrix_synapse_app_service_config_files` as discussed before, because that would break backward compatibility.

We can probably think of a different name though. One which is more consistent.

How about `matrix_synapse_app_service_config_files_final` or `matrix_synapse_app_service_config_files_combined`?